targets/targets.json

```diff
@@ -6336,7 +6336,14 @@
         "supported_form_factors": ["ARDUINO"],
         "inherits": ["MCU_NRF52832"],
         "release_versions": ["5"],
-        "device_name": "nRF52832_xxAA"
+        "device_name": "nRF52832_xxAA",
+        "config": {
```

Comment:
I think this can be defined as a fixed value and not a parameter controlled by the user.

Reply:
Changed to a 'macro'